.gitlab-ci.yml

```diff
@@ -1033,6 +1033,8 @@ workflow:
     allow_failure: true
 
 .on_trace_agent_changes_or_manual:
+  - <<: *if_mergequeue
+    when: never
```

Comment:
It is the same right ?
```suggestion
    - !reference [.except_mergequeue]
```

Reply:
It's the same but I prefer "natural" yaml references when in the same file. There is no real rule about this though..